aea/configurations/base.py

```diff
@@ -1178,6 +1184,12 @@ class SkillConfig(ComponentConfiguration):
     default_configuration_filename = DEFAULT_SKILL_CONFIG_FILE
     package_type = PackageType.SKILL
 
+    FIELDS_ALLOWED_TO_UPDATE = [
+        "behaviours",
+        "handlers",
+        "models",
+    ]
```

Comment:
actually; only the "args" field nested under these fields is configurable...

Reply:
it's done with schema validation, and actually  works